_includes/head.html

```diff
@@ -14,25 +14,27 @@
   <noscript><link rel="stylesheet" href="{{ "/assets/css/tooltip.css" | relative_url }}?v={{ site.time | date:'%s' }};"></noscript>
 
   <script> if (document.cookie.includes('logged_in=true') && window.location.href == 'https://plausible.io/') window.location.pathname = '/sites'</script>
-  <script
-    defer
-    data-domain="plausible.io"
-    src="https://plausible.io/js/script.manual.js"
-  ></script>
-  <script>
-    window.plausible = window.plausible || function() { (window.plausible.q = window.plausible.q || []).push(arguments) };
-    var props = {
-      {% if page.author %}author: "{{ page.author }}",{% endif %}
-      browser_language: navigator.language || navigator.userLanguage
-    };
-    window.plausible('pageview', {props: props});
-    window.addEventListener('pageshow', function(event) {
-      if (event.persisted) {
-        window.plausible('pageview', {props: props});
+  <script {% if page.author %}data-author="{{page.author}}{% endif %}">
```

Comment:
Noticed a loose `"` sign
```suggestion
  <script {% if page.author %}data-author="{{page.author}}"{% endif %}>
```